source/Irrlicht/CSkinnedMesh.cpp

```diff
@@ -447,12 +453,12 @@ scene::SCPUSkinMeshBuffer *CCPUSkinnedMesh::addMeshBuffer()
 }
 
 
-void CCPUSkinnedMesh::addMeshBuffer(SCPUSkinMeshBuffer* buf)
+void CCPUSkinnedMesh::addMeshBuffer(ICPUMeshBuffer* buf)

```

Comment:
CCPUSkinnedMesh should ONLY contain SCPUSkinMeshBuffers!!!

Reply:
And there's appropriate dynamic_cast